Documentation/ApiOverview/SymfonyExpressionLanguage/Index.rst

```diff
@@ -41,9 +41,10 @@ The following section explain how to add variables and functions.
 Registering new provider within an extension
 ============================================
 
-There has to be a provider, no matter whether variables or functions should be provided.
+There has to be a provider, no matter whether variables or functions shall be provided.
```

Comment:
I'm not sure but I kind of remember that "shall" should (heh) be avoided ...

Reply:
==> will.